packages/documentation-site/patternfly-docs/content/demos/Compass/CompassReactFlowDemo.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { memo, useCallback, useRef } from "react";
+import {
+  Background,
+  ReactFlow,
+  useNodesState,
+  useEdgesState,
+  addEdge,
+  useReactFlow,
+  ReactFlowProvider,
+  Handle,
+  Position,
+} from "@xyflow/react";
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardTitle,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Divider,
+  Title,
+  Badge,
+  Button,
+} from "@patternfly/react-core";
+import "@xyflow/react/dist/style.css";
+import "./reactFlowOverrides.css";
+
+const PFCardNode = memo(
+  ({
+    data,
+    isConnectable,
+  }: {
+    data: any;
+    isConnectable: boolean;
+  }) => {
+    return (
+      <>
+        <Handle
+          type="target"
+          position={Position.Left}
+          isConnectable={isConnectable}
+          style={{ background: "#555" }}
+        />
+        <div style={{ minWidth: "300px", maxWidth: "400px" }}>
+          <Card isFullHeight>
+            <CardTitle>
+              <div
+                style={{
+                  display: "flex",
+                  justifyContent: "space-between",
+                  alignItems: "center",
+                }}
+              >
+                <Title headingLevel="h4" size="lg">
```

Comment:
```suggestion
                <Title headingLevel="h2" size="lg">
```

We're going from an h1 to h4 headings in the demo, so just need to make these h2

Reply:
Was looking at the wrong demo for this comment, what you had with the h4 was correct 🙊